ui/app/models/database/connection.js

```diff
@@ -0,0 +1,165 @@
+import Model, { attr } from '@ember-data/model';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+import fieldToAttrs, { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+const AVAILABLE_PLUGIN_TYPES = [
+  {
+    value: 'mongodb-database-plugin',
+    displayName: 'MongoDB',
+    fields: [
+      { attr: 'name' },
+      { attr: 'plugin_name' },
+      { attr: 'password_policy' },
+      { attr: 'username', group: 'pluginConfig' },
+      { attr: 'password', group: 'pluginConfig' },
+      { attr: 'connection_url', group: 'pluginConfig' },
+      { attr: 'write_concern' },
+      { attr: 'creation_statements' },
+    ],
+  },
+  {
+    value: 'mongodbatlas-database-plugin',
```

Comment:
self: remove this before launch lest we disappoint people

Reply:
also: update the default form values to all possible values so that non-standard plugins could still be supported?